docs/examples/couchbase.md

```diff
@@ -0,0 +1,236 @@
+<!--[metadata]>
++++
+title = "Dockerizing a Couchbase service"
+description = "Dockerizing a Couchbase service"
+keywords = ["docker, example, package installation, networking, couchbase"]
+[menu.main]
+parent = "smn_applied"
++++
+<![end-metadata]-->
+
+# Dockerizing a Couchbase service
+
+This example shows how to start a [Couchbase](http://couchbase.com) server using Docker Compose, configure it using its [REST API](http://developer.couchbase.com/documentation/server/4.0/rest-api/rest-endpoints-all.html), and query it.
+
+Couchbase is an open source, document-oriented NoSQL database for modern web, mobile, and IoT applications. It is designed for ease of development and Internet-scale performance.
+
+## Start Couchbase server
+
+Couchbase Docker images are published at [Docker Hub](https://hub.docker.com/_/couchbase/).
+
+Start Couchbase server as:
+
+```
+docker run -d --name db -p 8091-8093:8091-8093 -p 11210:11210 couchbase
+```
+
+The purpose of each port exposed is explained at [Couchbase Developer Portal - Network Configuration](http://developer.couchbase.com/documentation/server/4.1/install/install-ports.html).
+
+Logs can be seen as:
+
+```
+docker logs mycouchbase
+Starting Couchbase Server -- Web UI available at http://<ip>:8091
+```
+
+> **Note**:
+> We're assuming Couchbase is running on a Docker Machine. The IP address of 
```

Comment:
We're trying to avoid "We" in the documentation. I think this can be written a bit more general.

Perhaps something like;

```Markdown
> **Note**: The examples on this page assume that the Docker Host
> is reachable on `192.168.99.100`. Substitute `192.168.99.100` with
> the actual IP address of your Docker Host.  If you're running
> Docker using Docker machine, you can obtain the IP address
> of the Docker host using `docker-machine ip <MACHINE-NAME>`.
```